<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleCameraSettingsWidget</class>
 <widget class="QWidget" name="MultipleCameraSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QnGeneralCameraSettingsWidget" name="generalCameraSettingsWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRecording">
      <attribute name="title">
       <string>Recording</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QnCameraScheduleWidget" name="cameraScheduleWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="expertTab">
      <attribute name="title">
       <string>Expert</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QnAdvancedSettingsWidget" name="expertSettingsWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnCameraScheduleWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/properties/camera_schedule_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnAdvancedSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/properties/expert_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnGeneralCameraSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/properties/general_camera_settings_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>buttonClicked(QAbstractButton*)</slot>
 </slots>
</ui>
